app/routes/sets.js

```diff
@@ -1,13 +1,27 @@
 import Route from '@ember/routing/route';
 import { service } from '@ember/service';
+import { hash } from 'rsvp';
 
 export default class SetsRoute extends Route {
   @service store;
 
   async model() {
-    return this.store.query('cardCycle', {
+    let cycles = await this.store.query('cardCycle', {
```

Comment:
```suggestion
    let cycles = this.store.query('cardCycle', {
```

No need to await if we're passing it to `hash`.